spec/lib/sentry/processor/filter_request_body_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Sentry::Processor::FilterRequestBody do
+  let(:client) { double('client') }
+  let(:processor) { Sentry::Processor::FilterRequestBody.new(client) }
+  let(:result) { processor.process(data) }
+
+  let(:data) do
```

Comment:
data is not the best name here since you're also focused on the `data` key in the `request`